### DISCUSSION TITLE
hdfs读取异常

### DISCUSSION THREAD

**Discussion by inurfuture** [alibaba/GraphScope#1181] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42594720/146478606-20539a8e-fd1b-4898-ac55-1eb61f1bc49c.png)
异常信息如上，参考文档中的载图-loader变种里的方法（ds3 = Loader("hdfs:///datafiles/group.e", host='localhost', port='9000', extra_conf={'conf1': 'value1'})）

**Comment by sighingnow** [comment]
Seems broken in v0.9.0, will be fixed ASAP.

**Reply by wenyuanyu** [reply]
Maybe we can refine CI to catch such cases...

**Comment by yecol** [comment, marked as answer]
fixed in #1229
